plugins/modules/kdeconfig.py

```diff
@@ -0,0 +1,277 @@
+#!/usr/bin/python
+
+# Copyright (c) 2023, Salvatore Mesoraca <s.mesoraca16@gmail.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+
+DOCUMENTATION = r'''
+---
+module: kdeconfig
+short_description: Manage KDE configuration files
+version_added: "6.5.0"
+description:
+  - Add or change individual settings in KDE configuration files.
+  - It uses B(kwriteconfig) under the hood.
+
+options:
+  path:
+    description:
+      - Path to the config file. If the file does not exist it will be created.
+    type: path
+    required: true
+  kwriteconfig_path:
+    description:
+      - Path to the kwriteconfig executable. If not specified, Ansible will try
+        to discover it.
+    type: path
+  values:
+    description:
+      - List of values to set.
+    type: list
+    elements: dict
+    suboptions:
+      group:
+        description:
+          - The option's group. One between this and I(groups) is required.
+        type: str
+      groups:
+        description:
+          - List of the option's groups. One between this and I(group) is required.
+        type: list
+        elements: str
+      key:
+        description:
+          - The option's name.
+        type: str
+        required: true
+      value:
+        description:
+          - The option's value. One between this and I(bool_value) is required.
+        type: str
+      bool_value:
+        description:
+          - Boolean value.
+          - One between this and I(value) is required.
+        type: bool
+    required: true
+  backup:
+    description:
+      - Create a backup file.
+    type: bool
+    default: false
+extends_documentation_fragment:
+  - files
+  - attributes
```

Comment:
Sorry, should have been
```suggestion
  - community.general.attributes
```

Reply:
Yep, fixed. Thanks